irodsConnector/resources.py

```diff
@@ -104,7 +100,7 @@ def get_resource_children(self, resc: irods.resource.iRODSResource) -> list:
             children.extend(self.get_resource_children(child))
         return resc.children + children
 
-    def resources(self, update = False) -> dict:
+    def resources(self, update: bool=False) -> dict:
```

Comment:
It's rather confusing, but with type hints you need spaces around the "=", but without you don't...

```suggestion
    def resources(self, update: bool = False) -> dict:
```